src/lib.rs

```diff
@@ -182,20 +169,16 @@ pub mod tcp {
     pub use libp2p_tcp::*;
```

Comment:
Sorry if that was already discussed, but could you explain again why the `#[cfg_attr(docsrs, doc(cfg(feature = "_")))]` flag is not removed for the crates that have extra features (like tcp/ dns / ..)?

Reply:
It hasn't been discussed, well spotted!

I think those should be removed too, yes! Line 166 here for example.

Reply:
In other words, every manually added `#[cfg_attr(docsrs, doc(cfg(feature = "..")))]` should be removed?